### PR TITLE
[FIX] Training error when batch size is 1

### PR DESCRIPTION
**Pull request by cih9088**
### Summary
When the last batch size is 1, training batch normalization is a nonsense. We should drop it instead.

### Changes
- `drop_last=True` if the last batch size is 1.

### CODE REVIEW

**Review by sungmanc — APPROVED** [review]

**Review by goodsong81 — COMMENTED** [review]
@JihwanEom Could you take a look? I remember that there was a bug fix related to drop_last option. Please double-check :)

**Comment by JihwanEom** [comment]
> @JihwanEom Could you take a look? I remember that there was a bug fix related to drop_last option. Please double-check :)

It's okay for me (also passed h-label tests), the drop_last related issue was raised when 1 < sample_size < batch_size. LGTM!

**Review by JihwanEom — APPROVED** [review]
